### DISCUSSION TITLE
Id payments invoices

### DISCUSSION THREAD

**Discussion by Alex-MRX-777** [btcpayserver/btcpayserver#4284] [Technical Support]
tell me why the page has such a view (screen), and when accessing the api, I get this view "da23074fdaa3829734ffd7e3c629bbb790398bdaefcaf0e71e911636e97aeb53-0", what is this "-0"(I also saw "-1")?
[[](url)](https://chat.btcpayserver.org/files/j35hshy1ypya5j5rd9ef8bn64r/public?h=2RQgqO5yo4iVXnliYvVhbHFlB0Wof5ijPBK0waR5eSM)

**Comment by bolatovumar** [comment]
@Alex-MRX-777 could you provide more details? What API endpoint are you using?

**Reply by Alex-MRX-777** [reply]
I create an invoice, after I pay for it, and when I receive data from "Get invoice payment methods" then the transaction id has 2 characters at the end -(0-6), as they explained to me this number of confirmations, it was not bad to separate this data and make a new field "confirmation".

**Reply by Kukks** [reply]
> I create an invoice, after I pay for it, and when I receive data from "Get invoice payment methods" then the transaction id has 2 characters at the end -(0-6), as they explained to me this number of confirmations, it was not bad to separate this data and make a new field "confirmation".

It is not a confirmation, It is the index of output in the transaction that pays the invoice
